### PR TITLE
feat(*): add create and edit commands

### LINKED ISSUE #8: `helm edit`
Homebrew provides a nice little convenience tool called `brew edit` to help formula authors.

Could we provide something similar for `helm`? @gabrtv has some ideas about how we might implement this. Perhaps he can chime in here.

### PR DESCRIPTION
**Pull request by gabrtv**
This PR adds `helm create` and `helm edit` commands which make for a nice contributor workflow.

Note a lot of this code will have to be refactored once we move chart serialization/deserialization into the `model` package.

Example of `helm create`:

```console
$ helm create mychart
Copying /Users/gabriel/workspace/src/github.com/deis/helm/skel
Copying /Users/gabriel/workspace/src/github.com/deis/helm/skel/Chart.yaml
Copying /Users/gabriel/workspace/src/github.com/deis/helm/skel/README.md
Copying /Users/gabriel/workspace/src/github.com/deis/helm/skel/manifests
Copying /Users/gabriel/workspace/src/github.com/deis/helm/skel/manifests/example-pod.yaml
```

Example of `helm edit` which opens in `$EDITOR` and saves the results:

```console
$ helm edit mychart
--- # Chart.yaml
name: example-pod-2
home: http://github.com/deis/helm
version: 0.0.1
description: Example pod running Alpine Linux.
maintainers:
  - Gabe Monroy <gabe@deis.com>
details:
  This package provides a basic Alpine Linux image that can be used for basic
  debugging and troubleshooting. By default, it starts up, sleeps for a long
  time, and then eventually stops.
--- # end
--- # manifests/example-pod.yaml
apiVersion: v1
kind: Pod
metadata:
  name: alpine
spec:
  restartPolicy: Never
  containers:
  - name: alpine
  image: "alpine:3.2"
    command: ["/bin/sleep","9000"]
--- # end
```

Closes #8.

### CODE REVIEW

File: helm/action/create.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+	}
+
+	chartDir := path.Join(homeDir, "workspace", "charts", chartName)
+
+	// copy skeleton to chart directory
+	if err := copyDir(skeletonDir, chartDir); err != nil {
+		log.Die("failed to copy skeleton directory: %v", err)
+	}
+
+	return
```
Comment (mboersma) [inline helm/action/create.go]: An empty `return` here isn't needed, and it made me re-scan the function signature to see if there were actually named return variables.

File: helm/action/edit.go [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+		log.Die("could not open tempfile: %v", err)
+	}
+	defer os.Remove(f.Name())
+	f.Write(contents.Bytes())
+	f.Close()
+
+	openEditor(f.Name())
+	saveChart(chartDir, f.Name())
+
+	return
```
Comment (mboersma) [inline helm/action/edit.go]: As above, I found the unneeded `return` a bit confusing.

**Comment by mboersma** [comment]
This should close #8.

**Comment by technosophos** [comment]
Yeah, other than those few things @mboersma mentioned, this all looks good to me.

**Pull request merged**